db/migrate/20210519204247_move_miq_set_memberships.rb

```diff
@@ -0,0 +1,65 @@
+class MoveMiqSetMemberships < ActiveRecord::Migration[5.0]
+  class Relationship < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class MiqSetMembership < ActiveRecord::Base
+  end
+
+  def up
+    say_with_time("Converting set relationships to miq_set_memberships") do
+      # The `ancestry` column holds the id for the `relationship` record that
+      # references the miq_set the parent record contains the miq_set object
+      #
+      # Because these are trivial trees (1 parent with 1 level of children),
+      # the `miq_set_id` can be derived from the `resource_id` of the parent
+      miq_set_id = "(SELECT rel2.resource_id FROM relationships rel2 WHERE rel2.id = rels.ancestry::bigint)"
+
+      connection.execute(<<-SQL.squish)
+        INSERT INTO miq_set_memberships (member_type, miq_set_id, member_id, created_at, updated_at)
+          SELECT rels.resource_type, #{miq_set_id}, rels.resource_id, rels.created_at, rels.updated_at
```

Comment:
Super minor, but readability wise, I think member_type and member_id should be next to each other since they are a pair.

Reply:
oh, sure.  Not sure why I did it that way, but I agree.  👍

Reply:
Oh, that is why I didn't remember doing that... that was how this was when I got here...